src/engine/renderer/gl_shader.cpp

```diff
@@ -623,23 +695,23 @@ bool GLShaderManager::buildPermutation( GLShader *shader, int macroIndex, int de
 	return false;
 }
 
-void GLShaderManager::buildAll( )
+void GLShaderManager::buildAll()
 {
 	while ( !_shaderBuildQueue.empty() )
 	{
-		GLShader *shader = _shaderBuildQueue.front();
-		size_t numPermutations = 1 << shader->GetNumOfCompiledMacros();
+		GLShader& shader = *_shaderBuildQueue.front();
+		size_t numPermutations = 1 << shader.GetNumOfCompiledMacros();
 		size_t i;
 
 		for( i = 0; i < numPermutations; i++ )
 		{
-			buildPermutation( shader, i, 0 );
+			buildPermutation( &shader, i, 0 );
 		}
 
 		_shaderBuildQueue.pop();
 	}
 
-	ri.Printf( PRINT_DEVELOPER, "glsl shaders took %d msec to build\n", _totalBuildTime );
+	Log::Debug( "glsl shaders took %d msec to build\n", _totalBuildTime );
```

Comment:
That's going to print it unconditionnzlly which is going to be too verbose for users, use a Logger (Log.h) if you want to use Log